packages/ui-extensions/src/surfaces/admin/api/order-routing-rule/order-routing-rule.doc.ts

```diff
@@ -0,0 +1,20 @@
+import {ReferenceEntityTemplateSchema} from '@shopify/generate-docs';
```

Comment:
Are these docs ready to ship? Once this change gets merged in to the `unstable` branch, someone might generate the docs and cause this API to be published.

Reply:
Not at all 😓

Reply:
We should just remove these docfiles until we're ready to publish

Reply:
Will do! I think we weren't super sure what was the process here so this was a bit of a shot in the dark. If I understand correctly, we should:
- Commit the types so they get to unstable
- Work on the docs separately and merge them when ready

Would that be the expected workflow?

Is there a timeline as to when unstable becomes stable? Is there any sort of deadline we'd want to hit?